frontend/src/scenes/insights/InsightPageHeader.tsx

```diff
@@ -143,10 +143,6 @@ export function InsightPageHeader({ insightLogicProps }: { insightLogicProps: In
                                             >
                                                 Share or embed
                                             </LemonButton>
-                                        </>
-                                    )}
-                                    {insight.short_id && (
```

Comment:
Don't think there's a way that this condition (`insight.short_id`) and `hasDashboardItemId` aren't equal.